tests/scripts/test_label.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# shellcheck source=tests/scripts/config.source
+. "$(dirname "$(realpath "$0")")/config.source"
+
+require_min_pulp "3.10.0" || exit 0
```

Comment:
This is a nice primitive!
I would however move the `exit 0` into the function and accompany it with a `echo SKIPPING`.
Maybe with a special error code, we can tell the "test_scripts.py" that it was skipped instead of passed.